earthpy/spatial.py

```diff
@@ -275,24 +287,23 @@ def crop_image(raster, geoms, all_touched=True):
     Parameters
     ----------
     raster : rasterio object
-        The rasterio object to be cropped. Ideally this object is opened in a
-        context manager to ensure the file is properly closed.
-    geoms : geopandas object or list of polygons
-        Polygons are GeoJSON-like dicts specifying the boundaries of features
-        in the raster to be kept. All data outside of specified polygons
-        will be set to nodata.
-    all_touched : bool
-        From rasterio: Include a pixel in the mask if it touches any of the
+        The rasterio object to be cropped.
+    geoms : geopandas geodataframe or list of polygons
```

Comment:
The spatial polygon boundaries in GeoJSON-like dict format to be used to crop the image. All data outside
of the polygon boundaries will be set to nodata and/or removed from the image.